x86_64-unknown-intermezzos-gnu.json

```diff
@@ -3,15 +3,20 @@
 	"cpu": "x86-64",
 	"data-layout": "e-m:e-i64:64-f80:128-n8:16:32:64-S128",
 	"executables": true,
-        "linker-flavor": "gcc",
+	"linker": "gcc",
```

Comment:
Okay, thanks for looking into this! I'd _guess_ that the linker set in the cargo configs will probably take precedence, but I haven't looked at the source or anything to back that up.